### PR TITLE
Fix JS errors in editor-expand.js

### PR DESCRIPTION
**Pull request by joyously**
Fix the JS errors from the missing functions in editor-expand.js.
part of #53 

The old button function has a `classes` key, that is not used by the new function.

### CODE REVIEW

File: classicpress-editor.php [hunk 1]
```diff
@@ -38,10 +37,10 @@ public function __construct() {
 
 		// Ensure TinyMCE is loading from within this plugin, not core.
 		add_filter( 'includes_url', [ $this, 'filter_tinymce_includes_url' ], 10, 2 );
+		add_filter( 'script_loader_src', [ $this, 'filter_editor_expand_url' ], 10, 2 );
```
Comment (johnalarcon) [inline classicpress-editor.php:40]: This line (and the corresponding method) are the only changes in this file related to the change at hand.

**Review by johnalarcon — CHANGES_REQUESTED** [review]
The version bump is fine to accompany with this PR, but, the other edits to `classicpress-editor.php` should be changed back as they are unrelated to changing the `editor-expand.js` target file.

File: classicpress-editor.php (hunk 1 above)
Comment (joyously) [reply classicpress-editor.php:40]: Well, I was cleaning up from the unreviewed commits and reverts you did earlier.

Comment (johnalarcon) [reply classicpress-editor.php:40]: Point taken. They weren't reviewed, but, they were separate. :)

**Review by johnalarcon — APPROVED** [review]
I appreciate your moving this. This is a crap title, though – it doesn't indicate what is being accomplished. Please don't use titles like this in the future.

**Review by johnalarcon — APPROVED** [review]

**Review by johnalarcon — APPROVED** [review]

**Pull request merged**